src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -288,14 +365,14 @@ void PacketDataReporter::OnResource(ResourceType type, const ResourceData & data
         {
             OnOperationalSrvRecord(data.GetName(), srv);
         }
-        else if (mDiscoveryType == DiscoveryType::kCommissionableNode)
+        else if (mDiscoveryType == DiscoveryType::kCommissionableNode || mDiscoveryType == DiscoveryType::kCommissionerNode)
```

Comment:
a helper could be used here - I think this ||== is added in many places and is fairly long.